include/swift/AST/ASTWalker.h

```diff
@@ -164,7 +164,7 @@ class ASTWalker {
   virtual Pattern *walkToPatternPost(Pattern *P) { return P; }
 
   /// walkToDeclPre - This method is called when first visiting a decl, before
-  /// walking into its children.  If it returns false, the subtree is skipped.
+  /// walking into its children.  If it returns true, the subtree is skipped.
```

Comment:
Are you sure about these? That seems wrong when the default behavior is to return `true`.

Reply:
I... don't know what I was thinking. You're absolutely right, this makes no sense. I'll remove this commit.